Task10_Auth/src/main/java/reseponsesDTO/UsersDto.java

```diff
@@ -43,4 +67,19 @@ public String getZipCode() {
     public void setZipCode(String zipCode) {
         this.zipCode = zipCode;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof UsersDto)) return false;
+        UsersDto usersDto = (UsersDto) o;
+        if (getName() == null || getUserSex() == null) return false;
```

Comment:
It makes sense to do the same for sex field too

Reply:
To do what?

Reply:
Will discuss in skype

Reply:
done,
added equals for all fields
and there is an additional check that required fields are not null